src/gtk3-porting-examples/InfoSlicer.md

```diff
@@ -0,0 +1,289 @@
+Ticket of this port is [here](http://bugs.sugarlabs.org/ticket/3742), with some
+useful information that I'm using on the porting and to keep tracking
+this port.
+
+
+Remove hippo
+============
+
+First of all, I had to remove the *hippo* related things and the
+[sugar-port](Development_Team/sugar-port "wikilink")
```

Comment:
Please remove this link, since the page is now deprecated

Reply:
done.